### PR TITLE
Add tags to Functions "hello world" project files

### PR DESCRIPTION
**Pull request by jskeet**
cc @labtopia

### CODE REVIEW

File: functions/helloworld/HelloWorldFSharp/HelloWorldFSharp.fsproj [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (jskeet) [inline functions/helloworld/HelloWorldFSharp/HelloWorldFSharp.fsproj:1]: Note: the "-" here is because there was a leading BOM.

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Comment by jskeet** [comment]
(Will wait for Katie to comment before merging.)

**Review by labtopia — APPROVED** [review]
LGTM.

**Pull request merged**